### DISCUSSION TITLE
SqliteStudio 34.xx versions are slow

### DISCUSSION THREAD

**Discussion by koka55github** [pawelsalawa/sqlitestudio#4914] [Q&A]
My 3.3.3 works perfect since year(s?) but all the 3.4.xx versions (installer or portable, 32 or 64 bit) are unacceptable slow.
Conditions: ca. 1000 records with 200kbyte-1Mbyte bulk field (pictures). (Windows 11.)
Version 3.3.3: <1sec but Ver. 3.4.xx: ca. 30 sec (and huge CPU power last)!
Thanks for any help!

**Comment by pawelsalawa** [comment]
This is the eternal argument between people, who want full field values to be read at once vs people who experience this slowness and want field values to be truncated initially (and loaded fully only when edited). It has been a headache for a long time.

Please try setting this setting to some lower value (start with 10, then try with bigger values if it still works fine):
![image](https://github.com/pawelsalawa/sqlitestudio/assets/481846/388209f2-29f7-4684-bc23-dd0e21344d07)

**Comment by koka55github** [comment]
Thanks for the answer but it not works.
With 10 rows per page is OK but with 100 the same effect as with 1000. :(
But anyway, the Version 3.3.3 works fine! I will remain it, don't will update. :)
